xml/js/src/test/scala/scala/xml/XMLTest.scala

```diff
@@ -0,0 +1,877 @@
+  //         Elem(null, "title", e, sc, Text("Foundations of Programming Languages")))))
+
+  //   assertTrue(
+  //     (parsedxml2 \ "_" \ "_") sameElements List(
+  //       Elem(null, "author", e, sc, Text("Peter Buneman")),
+  //       Elem(null, "author", e, sc, Text("Dan Suciu")),
+  //       Elem(null, "title", e, sc, Text("Data on ze web")),
+  //       Elem(null, "author", e, sc, Text("John Mitchell")),
+  //       Elem(null, "title", e, sc, Text("Foundations of Programming Languages"))))
+
+  //   assertTrue(
+  //     (parsedxml2 \ "_" \ "author") sameElements List(
+  //       Elem(null, "author", e, sc, Text("Peter Buneman")),
+  //       Elem(null, "author", e, sc, Text("Dan Suciu")),
+  //       Elem(null, "author", e, sc, Text("John Mitchell"))))
+
+  //   assertTrue((parsedxml2 \ "_" \ "_" \ "author").isEmpty)
+  // }
+
+  // @UnitTest
+  // def xpathDESCENDANTS = {
+  //   assertTrue(
+  //     (parsedxml2 \\ "author") sameElements List(
+  //       Elem(null, "author", e, sc, Text("Peter Buneman")),
+  //       Elem(null, "author", e, sc, Text("Dan Suciu")),
+  //       Elem(null, "author", e, sc, Text("John Mitchell"))))
+
+  //   assertTrue(
+  //     (parsedxml2 \\ "title") sameElements List(
+  //       Elem(null, "title", e, sc, Text("Data on ze web")),
+  //       Elem(null, "title", e, sc, Text("Foundations of Programming Languages"))))
+
+  //   assertEquals("<book><author>Peter Buneman</author><author>Dan Suciu</author><title>Data on ze web</title></book>",
+  //     (parsedxml2 \\ "book") { n: Node => (n \ "title") xml_== "Data on ze web" } toString)
+
+  //   assertTrue(
+  //     ((NodeSeq.fromSeq(List(parsedxml2))) \\ "_") sameElements List(
+  //       Elem(null, "bib", e, sc,
+  //         Elem(null, "book", e, sc,
+  //           Elem(null, "author", e, sc, Text("Peter Buneman")),
+  //           Elem(null, "author", e, sc, Text("Dan Suciu")),
+  //           Elem(null, "title", e, sc, Text("Data on ze web"))),
+  //         Elem(null, "book", e, sc,
+  //           Elem(null, "author", e, sc, Text("John Mitchell")),
+  //           Elem(null, "title", e, sc, Text("Foundations of Programming Languages")))),
+  //       Elem(null, "book", e, sc,
+  //         Elem(null, "author", e, sc, Text("Peter Buneman")),
+  //         Elem(null, "author", e, sc, Text("Dan Suciu")),
+  //         Elem(null, "title", e, sc, Text("Data on ze web"))),
+  //       Elem(null, "author", e, sc, Text("Peter Buneman")),
+  //       Elem(null, "author", e, sc, Text("Dan Suciu")),
+  //       Elem(null, "title", e, sc, Text("Data on ze web")),
+  //       Elem(null, "book", e, sc,
+  //         Elem(null, "author", e, sc, Text("John Mitchell")),
+  //         Elem(null, "title", e, sc, Text("Foundations of Programming Languages"))),
+  //       Elem(null, "author", e, sc, Text("John Mitchell")),
+  //       Elem(null, "title", e, sc, Text("Foundations of Programming Languages"))))
+  // }
```

Comment:
Again, if no plan to support these tests for the ScalaJS test suite, then just delete instead of comment out?